ReactiveCocoa/UIKit/iOS/UIRefreshControl.swift

```diff
@@ -13,8 +13,19 @@ extension Reactive where Base: UIRefreshControl {
 		return makeBindingTarget { $0.attributedTitle = $1 }
 	}
 
-	/// A trigger that sends `next` when the user performs a refresh.
-	public var refresh: Signal<Void, NoError> {
-		return trigger(for: .valueChanged)
+	/// The action to be triggered when the refresh control is refreshed. It
+	/// also controls the enabled state of the refresh control.
+	public var refreshed: CocoaAction<Base>? {
```

Comment:
It was originally called refresh. I think that is the best name for it but a change was requested.

I used past a past tense command name to match the past tense command names found on other types in ReactiveCocoa.